v2/azure-arm.yaml

```diff
@@ -130,8 +130,45 @@ typeFilters:
 # weird json or by handling them differently in the generator.
 anyTypePackages:
 
+
+# 
+# Type Transformers are used to make precision edits to the object model we've loaded from the 
+# original OpeanAPI/Swagger files.
+#
+# We need these because there is great variability in how different product groups define their 
+# APIs, and because some of the common constructs that are used can't be directly handled in Go.
+#
+# The available properties fall into two groups - the first used to select which type needs to be 
+# transformed, and the second to define the required transformation.
+#
+# Selection Properties
+#
+# group    - select types or properties in the specified group or groups
+# version  - select types or properties in the specified version or version(s)
+# name     - select types or properties with the specified name or name(s)
+# property - select properties with the given name or name(s)
+#
+# All of these properties are matchers:
+#
+# o They wllow multiple values, semicolon (;) separated
```

Comment:
```suggestion
# o They allow multiple values, semicolon (;) separated
```

Reply:
At first, I thought that `o` was a character there by mistake and not a bullet point

Reply:
Normally when I write bullet points like this, I indent like so:

``` go
#
# All of these properties are matchers:
#
#   o   They wllow multiple values, semicolon (;) separated
```

which makes it more obvious. However, our ASO linting objects, so this is my compromise.